bokehjs/src/lib/models/widgets/spinner.ts

```diff
@@ -6,10 +6,14 @@ import {bk_input} from "styles/widgets/inputs"
 
 const {floor, max, min} = Math
 
-function _get_sig_dig(num: number): number { // get number of digits
-  if (floor(num) !== num)
-    return num.toFixed(16).replace(/0+$/, '').split(".")[1].length
-  return 0
+function _get_sig_dig(num: number | null): number | null { // get number of digits
+  if (num) {
```

Comment:
`num != null` is the idiomatic approach, if you want to reduce `number |  null` to `number`.